Zotero/Scenes/Detail/PDF/ViewModels/PDFReaderActionHandler.swift

```diff
@@ -992,7 +992,8 @@ final class PDFReaderActionHandler: ViewModelActionHandler, BackgroundDbProcessi
 
         state.changes.insert(.selection)
 
-        guard let key = key else {
+        guard let key else {
+            state.changes.insert(.selectionDeletion)
```

Comment:
This line doesn't mean that selection has been deleted, but that selected annotation was unselected (by hand gesture, deletion or whatever reason). I'd move this to the end of  `update(objects:deletions:insertions:modifications:viewModel:)` where you have flag `selectionDeleted`.

Reply:
Check https://github.com/zotero/zotero-ios/pull/875/commits/e4f821f0e933122b71fd926b16252e8bf46f3f44